config.toml

```diff
@@ -1,11 +1,6 @@
 title = "Voter Registration"
 
-# Taxnomies used to categorize states and territories by registration type
-# ex. registration_types = [ 'by_mail', 'in_person', 'not_needed', 'online' ]
-[taxonomies]
-registration_type = "registration_types"
-
 [params]
 Owner = "USA.gov"
 Referrer_Short_Code = "voteusa"
-lang = "en-US"
+language = "english"
```

Comment:
The full language name is being used here because it maps to the current `data/registration_forms/` file names. This way there doesn't need to be any `weight` or extra logic to pull out the correct PDF for Spanish and English before the list of `Other languages`.